cf-deployment.yml

```diff
@@ -1082,6 +1082,10 @@ instance_groups:
           registration_interval: 20s
           uris:
           - api.((system_domain))/routing
+          health_check:
+            name: routing-api-health-check
+            script_path: "/var/vcap/jobs/routing-api/bin/routing_api_health_check"
+            timeout: 6s
```

Comment:
For posterity, can you please explain how/why `6s` was chosen as the timeout interval?

Reply:
@ctlong we based this off cloud controller timeout since we used the same healthcheck script to curl api endpoint. This value should be equal or more than [routing api health check timeout configured in the routing api](https://github.com/cloudfoundry/routing-release/blob/develop/jobs/routing-api/spec#L117).